app/scripts/lib/ppom/ppom-util.ts

```diff
@@ -195,7 +206,7 @@ async function findConfirmationBySecurityAlertId(
 
 async function validateWithController(
   ppomController: PPOMController,
-  request: JsonRpcRequest,
+  request: SecurityAlertsAPIRequest | JsonRpcRequest,
```

Comment:
In [`validateRequestWithPPOM`](https://github.com/MetaMask/metamask-extension/pull/26143/files#diff-56c892dd2b3324d3e12096b6e2280dafd007572a2c6797f8f6edc9a6f782b293R56) the output from `normalizePPOMRequest` is passed in as the `request` argument. 

Because the return type of `normalizePPOMRequest` includes `SecurityAlertsApiRequest`, the type of the request parameter here also needs to be widened.